src/client/components/Menu/Menu.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './menu.css';
+import menuItems from './menuItems';
+
+export default function Menu() {
+  return (
```

Comment:
If prefer extracting all logic such as mapping from the jsx/view. When its embedded its anonymous and saying: "figure it out yourself", Not terribly important here since we are dealing with almost nothing. 
I would also create a subcomponent called MenuItem.

Reply:
I have a question.
 <NavLink to={to} className={className} activeStyle={activeStyle}> 
 Should I use inline styling even though I already use a style sheet ?
 If I'm not mistaken, NavLink translates into an anchor tag <a> so I can style it with a:active instead of activeStyle.

Reply:
What about activeClassName instead? Then could import it

Reply:
Changed it to be the most simple implementation it can be.
Please have a look.

Reply:
Again, no need to battle over trifles.

But I would still do this instead :

```
export default function Menu() {
  return (
    <div className="menu">
      <ul className="menu-list">
         <MenuItem name="Plants" link="plants">
         <MenuItem name="Special Offers" link="special-offers">
         <MenuItem name="About" link="about-us">
         <MenuItem name="Contact" link="contact">
      </ul>
    </div>
  );
}
```

Here we reduce thinking to 0 seconds which happens to align with the component having a single responsibility principle.

Reply:
I have implemented the requested changes.
Tested it on my local, I got no console errors but I'm getting a linter error.
(parameter) name: any
'name' is missing in props validationeslintreact/prop-types